Vaadin/capstone-2024/src/main/java/com/example/application/views/collegebasketballprediction/CollegeBasketballPredictionsView.java

```diff
@@ -47,6 +47,8 @@ public class CollegeBasketballPredictionsView extends Composite<VerticalLayout>
     private List<Game> games; // list to hold all games
     private Grid<Game> grid; // the grid to display the games
 
+    private static List<String> conferenceNames;
```

Comment:
It's fine to define this conferenceNames = new ArrayList<>(); here for safety to avoid NPEs somewhere else if not checked.